tests/invalid_models_tests/test_backend_specific.py

```diff
@@ -27,7 +36,7 @@ class Model(models.Model):
         # Mock connection.validation.check_field method.
         v = connection.validation
         old_check_field = v.check_field
-        v.check_field = MethodType(mock, v)
+        v.check_field = MethodType(mock_check_field, v)
```

Comment:
could we use `with mock.patch.object(connections['default'].validation, 'check_field')` as done in the other test? I think it's a bit cleaner than the current approach (which was done when we didn't have mock available).